client/src/components/admin/FlagsPopup.tsx

```diff
@@ -3,14 +3,15 @@ import { getRequest } from '../../api';
 import { errorAlert, timeSince } from '../../util';
 
 interface FlagsPopupProps {
-    /* Function to modify the popup state variable */
```

Comment:
why did u get rid of my doc comment TT -- write doc comment for close and projectID pls